RevenueCatUI/Templates/V2/Variables/VariableHandlerV2.swift

```diff
@@ -372,15 +381,15 @@ extension VariablesV2 {
             return ""
         }
 
-        return localizations[period.periodLocalizationKey] ?? ""
+        if period.value > 1 {
+            return self.productPeriodWithUnit(package: package, localizations: localizations)
+        } else {
+            return localizations[period.periodLocalizationKey] ?? ""
+        }
     }
 
     func productPeriodAbbreviated(package: Package, localizations: [String: String]) -> String {
-        guard let period = package.storeProduct.subscriptionPeriod else {
-            return ""
-        }
-
-        return localizations[period.periodAbbreviatedLocalizationKey] ?? ""
+        return self.productPeriodAbbreviatedWithUnit(package: package, localizations: localizations)
```

Comment:
Just a nit: 
I see `productPeriodAbbreviatedWithUnit(package:,localization:)` is only called from here. Perhaps, to avoid the method jumping, the implementation of `productPeriodAbbreviatedWithUnit` could just go in this method `productPeriodAbbreviated`, saving the extra method `productPeriodAbbreviatedWithUnit`